dds_web/api/schemas/project_schemas.py

```diff
@@ -88,10 +88,7 @@ class Meta:
     pi = marshmallow.fields.String(
         required=True,
         allow_none=False,
-        validate=marshmallow.validate.And(
-            marshmallow.validate.Length(min=1, max=255),
-            dds_web.utils.contains_disallowed_characters,
-        ),
+        validate=marshmallow.validate.Email(),
```

Comment:
You also need to replace the lines here: https://github.com/zishanmirza/dds_web/blob/54eff2d3c739de1d628bb18a0c18c3d26bfa471c/dds_web/api/project.py#L571-L572 

with this: 
```
        # Add a new project to db
        try:
            new_project = project_schemas.CreateProjectSchema().load(p_info)
            db.session.add(new_project)
        except sqlalchemy.exc.OperationalError as err:
            raise DatabaseError(message=str(err), alt_message="Unexpected database error.")
```

Reply:
Sure, I can add a "try" statement.